cl.obo

```diff
@@ -1484,7 +1485,7 @@ relationship: capable_of GO:0002495 ! antigen processing and presentation of pep
 relationship: capable_of GO:0002540 ! leukotriene production involved in inflammatory response
 relationship: capable_of GO:0045730 ! respiratory burst
 relationship: develops_from CL:0000774 ! band form eosinophil
-relationship: has_part GO:0098537 ! lobed nucleus
+relationship: has_part GO:0098537 ! obsolete lobed nucleus
```

Comment:
Using an obsolete class, should make a ticket to fix

Reply:
Quick fix: The 3 places where this is used, you can substitute a compound entity (anonymous class expression) to say the same thing:
'has part some 'lobed nucleus'
->
'has part' some (nucleus and 'bearer or' some lobed) 
# lobed = PATO:0001979 (edited) 

@matentzn - do we need to fix before release?

Reply:
I dont really mind. It probbaly happens often, but we really should generally not use obsolete classes!

Reply:
Do we have a warning system in place for obsoletion imported terms?

Reply:
Unfortunately this is one of these few cases where we cant easily have a check, because there are too many errors in cl.owl, which means we restrict error checking to cl-edit.owl, but if we look at cl-edit.owl in isolation, we don't know which external classes are obsolete. I have made the robot report checking highly configurable now, but what I cant yet do is assign different error levels to different ontology flavours (cl.owl should have different things being flagged as an ERROR as cl-edit.owl etc because it includes external axioms).In any case, not easy. The most straight way is to do something like this: https://github.com/obophenotype/cell-ontology/pull/1132/files

Its hacky, and the problem deserves a deeper luck, but perhaps that sufficies for now.